views/index.py

```diff
@@ -250,3 +252,55 @@ def reset_with_token(token):
                            form=form,
                            token=token,
                            error=form.errors)
+
+
+@library.route('/wishlist', methods=['GET', 'POST'])
+def wishlist():
+    form = WishlistForm()
+    if form.validate_on_submit():
+        try:
+            new_wish_item = WishListItem(authors=form.authors.data,
+                                         title=form.title.data,
+                                         pub_year=form.year.data)
+            db.session.add(new_wish_item)
+            db.session.commit()
+            data = db.session.query(WishListItem).all()
+            wish_list_schema = WishListItemSchema(many=True)
+            output = wish_list_schema.dump(data)
+            return render_template('wishlist.html', form=form, wishes=output)
+        except:
```

Comment:
Please be specific about type of exception. You could specify multiple exception types if required.